packages/fftw3/fftw3.0.7.2/opam

```diff
@@ -15,7 +15,19 @@ build: [
       "--localstatedir=%{lib}%/fftw3/var"
       "--libdir=%{lib}%/fftw3/lib"
       "--includedir=%{lib}%/fftw3/include"
-      "--datarootdir=%{lib}%/fftw3/share"]
+      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
+  ["./configure" 
+      "CFLAGS=-I/usr/local/include"
+      "LDFLAGS=-L/usr/local/lib"
```

Comment:
I don't know if I could but these local paths are usually always added to the compiler/linker's search paths. My issue was that for some reason they got removed during the latest OSX update (El Capitan). Re-installing the toolchain brought them back apparently.

Generally speaking, I don't think it would hurt to add them unconditionally. I only vaguely remember of another case, a *BSD system I think, where they were not included by default.